### PR TITLE
Specific file includes, remove tags.

### PR DESCRIPTION
**Pull request by robb-broome**
I thought the files being included were causing load issues (turned out they weren't), but this gets rid of the tags as well.

### CODE REVIEW

**Comment by taylorzr** [comment]
Thanks Robb! Didn't realize I committed my tags.

File: groundhog_day.gemspec [hunk 1]
```diff
@@ -13,10 +13,8 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://github.com/taylorzr/groundhog_day"
   spec.license       = "MIT"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
+  spec.files         = ["README.md", "LICENSE.txt", "Rakefile"]
+  spec.files        += ['lib/groundhog_day.rb', 'lib/groundhog_day/date_extensions.rb', 'lib/groundhog_day/datetime_extensions.rb', 'lib/groundhog_day/time_extensions.rb','lib/groundhog_day/version.rb']
```
Comment (taylorzr) [inline groundhog_day.gemspec:17]: I prefer non-explicit so no one can forget to add any new file to this gemspec.

Comment (robb-broome) [inline groundhog_day.gemspec:17]: What are the .exe files that are included? Also, I'm unable to execute the ``git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }` locally - it fails. Wondering if this is causing problems when deploying this to an integration instance.

Comment (taylorzr) [inline groundhog_day.gemspec:17]: There aren't any executables included. This is just the default gem template. 

I can run that command locally without failure. If you can provide a stacktrace, I'll gladly look into it.

**Pull request closed without merging**